src/AppBundle/Controller/PageController.php

```diff
@@ -93,6 +94,17 @@ public function mouvementOrganisationAction()
     }
 
     /**
+     * @Route("/le-mouvement/la-carte", name="page_le_mouvement_la_carte")
+     * @Method("GET")
+     */
+    public function mouvementCarteAction()
+    {
+        return $this->render('page/la-carte.html.twig', [
+            'committees' => $this->getDoctrine()->getRepository(Committee::class)->findByStatus(Committee::APPROVED),
```

Comment:
Better but it's even better with a method like `->findApprovedCommittees()` or `->findForInteractiveMap()`.